### PR TITLE
[Gradle] Update to latest libraries

### PR DESCRIPTION
**Pull request by lernerb**
Pretty much just fixing the PR comments for the same PR in https://github.com/venmo/tooltip-view/pull/15, aka updating gradle to a more recent version.

### CODE REVIEW

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-2.2-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.5-all.zip
```
Comment (ronshapiro) [inline gradle/wrapper/gradle-wrapper.properties:6]: 4.7?

File: library/src/test/java/com/venmo/view/TooltipViewTest.java [hunk 2]
```diff
@@ -7,17 +7,15 @@
...
 
 import static com.venmo.view.Assertions.assertThat;
 import static org.robolectric.Shadows.shadowOf;
 
-@RunWith(RobolectricGradleTestRunner.class)
-@Config(constants = BuildConfig.class,
-        sdk = 21,
-        manifest = "build/intermediates/manifests/androidTest/debug/AndroidManifest.xml")
+@RunWith(RobolectricTestRunner.class)
+@Config(constants = BuildConfig.class, sdk = 21)
```
Comment (ronshapiro) [inline library/src/test/java/com/venmo/view/TooltipViewTest.java:18]: `sdk = 27`?

**Review by ronshapiro — APPROVED** [review]

File: gradle/wrapper/gradle-wrapper.properties (hunk 1 above)
Comment (lernerb) [reply gradle/wrapper/gradle-wrapper.properties:6]: Anything over 4.6 is currently throwing a gradle error during normal compilation, will investigate.

File: library/src/test/java/com/venmo/view/TooltipViewTest.java (hunk 2 above)
Comment (lernerb) [reply library/src/test/java/com/venmo/view/TooltipViewTest.java:18]: Will check it out 👍

**Pull request closed without merging**